application/aam-backend-service/src/main/kotlin/com/aamdigital/aambackendservice/export/usecase/DefaultRenderTemplateBatchUseCase.kt

```
@@ -0,0 +1,407 @@
+        }
+    }
+
+    private fun rewriteZipEntries(
+        zipBytes: ByteArray,
+        dataArray: ArrayNode,
+        targetFileNamePattern: String
+    ): ByteArray {
+        val entries =
+            try {
+                readZipEntries(zipBytes)
+            } catch (ex: Exception) {
```

detekt (1.23.8)

[warning] The caught exception is swallowed. The original exception could be lost.

(detekt.exceptions.SwallowedException)